sky/backends/backend_utils.py

```diff
@@ -2264,9 +2265,10 @@ def is_controller_up(
         # unnecessary costly refresh when the controller is already stopped.
         # This optimization is based on the assumption that the user will not
         # start the controller manually from the cloud console.
-        # The acquire_lock_timeout is set to 1 second to avoid hanging the
-        # command when multiple spot_launch commands are running at the same
-        # time. It should be safe to set it to 0 (try once to get the lock).
+        #
+        # The acquire_lock_timeout is set to 0 to avoid hanging the command when
+        # multiple spot_launch commands are running at the same time. It should
+        # be safe to set it to 0 (try once to get the lock).
```

Comment:
Q: why is it safe? If a call like `spot queue` tries to acquire the lock and fails to get it, what happens?